app/views/public/loans/show.html.slim

```diff
@@ -60,67 +65,64 @@
-        #summary.tab-pane
-          p.loan_description
-            = render_translation(@loan.details) || %Q{<span class="notfound">#{t :no_description_found}</span>}
-
-        // Disable timeline and repayments for now
-        / #timeline.tab-pane
-        /   = render 'project_events/timeline'
-        /
-        / - unless @repayments.empty?
-        /   #payments.tab-pane
-        /     .row
-        /       .col-md-7
-        /         h4 = t :payment_schedule
-        /         table.table.repayments
-        /           tr
-        /             th = t :status
-        /             th = t :due_date
-        /             th = t :amount
-        /           - @repayments.each do |pmt|
-        /             tr class=pmt.status
-        /               td
-        /                 b = t pmt.status
-        /               - if pmt.paid
-        /                 td = l pmt.date_paid, format: :long
-        /               - else
-        /                 td = l pmt.date_due, format: :long
-        /               td = pmt.amount_formatted
-
-        #other_loans.tab-pane
-          - if !@other_loans || @other_loans.count <= 1
-            p.empty = t :first_loan
+        li.nav-item
+          a.nav-link.active href="#details" = t :details
+        li.nav-item
+          a.nav-link href="#other_loans" = t :past_loans
+    #about_loan.tab-content.col-xs-12
+      #details.tab-pane
+        p.loan_description
+          - if @loan.details
+            = render_translation(@loan.details)
           - else
-            h4 = t :all_loans
-            table.table.other_loans.rowlink data-link="row"
-              - @other_loans.each do |loan|
-                tr class="#{'selected' if loan.id == @loan.id}"
-                  td
-                    - if loan.id == @loan.id
-                      = loan.signing_date_long
-                    - else
-                      a href=url_for([:public, loan]) = loan.signing_date_long
-                  td = format_currency(loan.amount, loan.currency)
-                  td = loan.status
-                  td.hidden-xs style="width: 50%" = render_translation loan.summary
+            .notfound = t :no_description_found
+      #other_loans.tab-pane
+        - if !@other_loans || @other_loans.count <= 1
+          p.empty = t :first_loan
+        - else
+          h4 = t :all_loans
+          table.table.other_loans.rowlink data-link="row"
+            - @other_loans.each do |loan|
+              tr class="#{'selected' if loan.id == @loan.id}"
+                td
+                  - if loan.id == @loan.id
+                    = loan.signing_date_long
+                  - else
+                    a href=public_loan_path = loan.signing_date_long
+                td = format_currency(loan.amount, loan.currency)
+                td = loan.status
+                td.hidden-xs style="width: 50%" = render_translation(loan.details)
 
+      // Tab content for timeline and repayments
```

Comment:
I get the motivation to leave code like this lying around, but it's all in git, I think we should delete big blocks like this when we aren't using them. We can always check out an old branch to see what it looked like if we reimplement this tab.

Reply:
Ok.